keyboards/essemi/fs026/readme.md

```diff
@@ -0,0 +1,24 @@
+# FS026
+
+Supported Hardware: *FS026 PDS*.
+
+This is a 17x6 keyboard with WS2812B key lighting.
+It's designed for a modular keyboard mount using an FSO26.
```

Comment:
Probably want to make sure you get this correct, given you're attempting to introduce support for it.
```suggestion
It's designed for a modular keyboard mount using an FS026.
```